src/osip-simulation-view/src/main/java/edu/kit/pse/osip/simulation/view/main/Drawer.java

```diff
@@ -13,7 +13,7 @@
     /**
      * The Drawer draws itself onto the GraphicsContext at its position.
      * @param context The context that the object draws itself onto
-     * @param time The current time in nanoseconds as supplied by ApplicationTimer
+     * @param timeDiff The difference in time in minutes since the last call of draw.
```

Comment:
Achso ok stimmt. Aber müsste der Paramter `time` in den Unterklassen nicht noch in `timeDiff` umbenannt werden?